### PR TITLE
fix zoom leading to `integrations:zoom` location

### PR DESCRIPTION
**Pull request by KATT**
I know very little of the Daily integration - can someone with knowledge please confirm that this looks right?

This area have been whack-a-mole for some time, a biiiit worried this might break something else.

### CODE REVIEW

File: lib/events/EventManager.ts [hunk 1]
```diff
@@ -117,17 +117,21 @@ export default class EventManager {
...
 
-    // First, create all calendar events. If this is a dedicated integration event, don't send a mail right here.
-    const results: Array<EventResult> = await this.createAllCalendarEvents(evt);
+    const results: Array<EventResult> = [];
     // If and only if event type is a dedicated meeting, create a dedicated video meeting.
     if (isDedicated) {
       const result = await this.createVideoEvent(evt);
       if (result.createdEvent) {
         evt.videoCallData = result.createdEvent;
+        evt.location = result.createdEvent.url;
```
Comment (KATT) [inline lib/events/EventManager.ts:126]: This seem to have been the missing piece.

**Review by pumfleet — APPROVED** [review]

File: lib/events/EventManager.ts [hunk 2]
```diff
@@ -138,6 +142,8 @@ export default class EventManager {
       };
     });
 
+    console.log({ referencesToCreate });
+
```
Comment (zomars) [inline lib/events/EventManager.ts:146]:
```suggestion
```

File: lib/events/EventManager.ts [hunk 3]
```diff
@@ -272,6 +278,8 @@ export default class EventManager {
   private createVideoEvent(event: Ensure<CalendarEvent, "language">): Promise<EventResult> {
     const credential = this.getVideoCredential(event);
 
+    console.log("createVideoEvent", { event });
+
```
Comment (zomars) [inline lib/events/EventManager.ts:282]:
```suggestion
```

**Review by zomars — APPROVED** [review]
looking gucci

**Pull request merged**